packages/appeals-service-api/src/spec/api-types.d.ts

```diff
@@ -518,7 +520,7 @@ export interface AppellantSubmission {
 	informedOwners?: boolean;
 	advertisedAppeal?: boolean;
 	appellantGreenBelt?: boolean;
-	updateDevelopmentDescription?: boolean;
+	updateDevelopmentDescription?: string;
```

Comment:
not related to a change I made, I guess someone forgor to run the type gen script?

Reply:
maybe we should auto run that when doing a migration locally